### PR TITLE
LPS-200508 refactor components from vulcan-api-impl with its own class name as service type

### PR DESCRIPTION
**Pull request by rcsdnj**
Removed from OSGi the GraphQLDTOContributorDataFetchingProcessor and LiferayMethodDataFetchingProcessor components and moved their initialization to the proper places

### CODE REVIEW

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java [hunk 1]
```diff
@@ -563,6 +572,23 @@ public void keyRemoved(
 				});
 		_graphQLRequestContextValidators = ServiceTrackerListFactory.open(
 			bundleContext, GraphQLRequestContextValidator.class);
+
```
Comment (rafaprax) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java:575]: @rcsdnj use the `ServiceTrackerListFactory.open()` instead to make the service tracker open a lazy operation.

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java (hunk 1 above)
Comment (rcsdnj) [reply modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java:572]: Done, could you check please?

**Comment by rafaprax** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Nice job @rcsdnj. Please forward this to the core-infra repo. Thx

**Pull request closed without merging**